WalletWasabi.Fluent/ViewModels/Wallets/Send/PrivacySuggestionsFlyoutViewModel.cs

```diff
@@ -68,10 +68,6 @@ await foreach (var suggestion in suggestions)
 		{
 			Logger.LogWarning("Computing privacy suggestions timed out.");
 		}
-		finally
```

Comment:
I think this was actually good to have here (without `cancellationTokenSource.Cancel();`) because then you make sure that the ring disappears even when an exception is thrown.

Reply:
I mean it's probably not necessary but probably better. So feel free to ignore, if you disagree.

Reply:
True... I will revert it.

Reply:
No, I agree with you perfectly, and the change makes sense. I deleted the finally block in rush.